server/routes/liteCategories.js

```diff
@@ -12,9 +12,37 @@ const joi = baseJoi.extend(dateExtend)
 
 /**
  * 'Lite' categorisation is simply a P-Nomis style workflow for any cats other than B/C/D/I/J.
- * This renders it unnecessary to go to P-Nomis for category assessments at all.
+ * This renders it unnecessary to go to P-Nomis for category assessments at all, hopefully!
  */
 
+const getPrisonList = (prisonListFromApi, current) => [
+  { value: '' },
+  ...prisonListFromApi.map(p => ({
+    value: p.agencyId,
+    text: sanitisePrisonName(p.description),
+    selected: current === p.agencyId,
+  })),
+]
+
+const getCommitteeList = current => [
+  { value: 'OCA', text: 'OCA', selected: current === 'OCA' },
+  { value: 'REVIEW', text: 'Review', selected: current === 'REVIEW' },
+  { value: 'RECP', text: 'Reception', selected: current === 'RECP' },
+  { value: 'SECUR', text: 'Security', selected: current === 'SECUR' },
+  { value: 'GOV', text: 'Governor', selected: current === 'GOV' },
+]
+
+const getCatList = current => [
+  { value: 'U', text: 'Unsentenced', selected: current === 'U' },
+  { value: 'Z', text: 'Unclass', selected: current === 'Z' },
+  { value: 'A', text: 'Cat A', selected: current === 'A' },
+  { value: 'E', text: 'Cat A Ex', selected: current === 'E' },
+  { value: 'H', text: 'Cat A Hi', selected: current === 'H' },
+  { value: 'P', text: 'Prov A', selected: current === 'P' },
+  { value: 'R', text: 'Fem Closed', selected: current === 'R' },
+  { value: 'T', text: 'Fem Open', selected: current === 'T' },
```

Comment:
Is this used anywhere else?  Just wondering if an enum would help

Reply:
I just refactored these from 2 copies of select values - one in each form html page. I could create an enum but this takes a parameter which makes it a little different?